pages/common/popeye.md

```diff
@@ -0,0 +1,20 @@
+# popeye
+
+> Utility that reports potential issues with your Kubernetes deployment manifests and configuration.
```

Comment:
Perhaps this will read better?
```suggestion
> Utility that reports potential issues with Kubernetes deployment manifests and configuration files.
```

Reply:
I will delete part `.. and configuration files`, cause it's actually just manifests in the game.